WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormSection.SettingsRow+Visibility.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import Yosemite
+
+extension ProductFormSection.SettingsRow {
```

Comment:
did you mean moving this extension to `ProductFormTableViewModel.swift` file where the `ProductFormSection.SettingsRow` enum is defined? if so, it sounds like a good idea since the extension is pretty small and I moved it in https://github.com/woocommerce/woocommerce-ios/pull/2217/commits/d74e10cc7965c8a3df84d789f4f7adc80cd11776

lemme know if I misunderstood the comment!

Reply:
That's correct 😃 👍